src/css/gamelab.css

```diff
@@ -22,6 +23,10 @@
   margin: 0px;
 }
 
+.spritelab-hide{
+  border: 0px !important;
```

Comment:
Is there a way to accomplish this without !important? I think !important is often a CSS code smell

Reply:
This is mostly relevant for the 'tool-icon' class that I'm hiding because the class 'tool-icon' has a border associated with it. Since we are hiding on some of the tool-icons, I decided to use !important as a way to make sure that border spacing is available for the tool-icons that are not hidden and border-spacing is zero for the items that are otherwise hidden. I don't know of another way to do that while still using the pattern of hiding items via css.

Reply:
Would a multi-class selector work?
```
.tool-icon {border: something}
.tool-icon.spritelab-hide {border: 0px}
```

Reply:
For the specific case of the tool-icon, I think that would definitely work. But I also can't think of a scenario where we'd want to hide a tool, but still have a border. As far as I know, with the multi-class selector, I'd have to call out each class that needs to have a zero-border, whereas with !important, I know that it will cover all classes to which I add '.spritelab-hide'. The benefit of this would be if more tools are hidden in restricted mode in the future - with !important, we wouldn't need to check if there is a border conflict and subsequently potentially add the multi-class selector.